.env.template

```diff
@@ -0,0 +1,41 @@
+# This is a sample .env file for use in local development.
+# Duplicate this file as .env in the root of the project
+# and update the environment variables to match your
+# desired config
```

Comment:
I guess we will assume that most of people working on this will know that $ has to be scaped in .env files. If not we could had a comment